app/assets/stylesheets/content/_in_place_editing.sass

```diff
@@ -10,8 +10,10 @@
     *
       cursor: wait!important
   &.preview
-    button, .help
+    button
```

Comment:
@0xF013 I'd like to get rid of these custom overrides as much as possible. Can we not use `ng-hide` instead?